lms/djangoapps/discussion_api/api.py

```diff
@@ -718,6 +718,50 @@ def get_thread(request, thread_id):
     return serializer.data
 
 
+def get_response_comments(request, comment_id, page, page_size):
+    """
+    Return the list of comments for the given thread response.
+
+    Arguments:
+
+        request: The django request object used for build_absolute_uri and
+          determining the requesting user.
+
+        comment_id: The id of the comment/response to get child comments for.
+
+        page: The page number (1-indexed) to retrieve
+
+        page_size: The number of comments to retrieve per page
+
+    Returns:
+
+        A paginated result containing a list of comments
+
+    """
+    try:
+        cc_comment = Comment(id=comment_id).retrieve()
+        cc_thread, context = _get_thread_and_context(request, cc_comment["thread_id"])
+        if cc_thread["thread_type"] == "question":
+            thread_responses = cc_thread["endorsed_responses"] + cc_thread["non_endorsed_responses"]
```

Comment:
Ignore that.  I see we're looking for a particular comment to get the children of, not returning all of them.  I guess the only suggestion I'd make is that instead of building a new list containing the contents of the endorsed and non-endorsed responses, we could save some memory by using `thread_responses = itertools.chain(endorsed, non_endorsed)`.